packages/nbmodel/src/ymodel.ts

```diff
@@ -293,6 +293,28 @@ export class YCodeCell
   getOutputs(): Array<nbformat.IOutput> {
     return this.outputs;
   }
+  /**
+   * Create a new YRawCell that can be inserted into a YNotebook
+   */
+  public static create(): YCodeCell {
+    const ymodel = new Y.Map();
+    const ysource = new Y.Text();
+    ymodel.set('source', ysource);
+    ymodel.set('metadata', {});
+    ymodel.set('cell_type', this.prototype.cell_type);
+    ymodel.set('execution_count', this.prototype.execution_count);
+    return new this(ymodel);
+  }
+  /**
+   * Create a new YRawCell that works standalone. It cannot be
+   * inserted into a YNotebook because the Yjs model is already
+   * attached to an anonymous Y.Doc instance.
+   */
+  public static createStandalone(): YCodeCell {
```

Comment:
This is superfluous as the same static method is defined by `YBaseCell`. We probably never have to change this method. I think this should be removed to avoid duplication.

Reply:
I have finally gone to removing the additional factory methods.